src/libs/fileDownload/FileUtils.ts

```diff
@@ -76,6 +79,11 @@ function showCameraPermissionsAlert() {
                 text: Localize.translateLocal('common.settings'),
                 onPress: () => {
                     Linking.openSettings();
+                    // In case of ios, app reload when we enable camera permission from settings
+                    // we are saving last screen so we can navigate to it after app reload
+                    if (getPlatform() === CONST.PLATFORM.IOS && screenName) {
```

Comment:
Don't use getPlatform function. Let's create a new .ios file

Reply:
created FileUtils.ios.ts file

Reply:
@ravindra-encoresky Ops, Don't need to create a new FileUtils.ios.ts. WƯe should introduce a new function, this function only be available on .ios file, and this func will be empty on other platforms. Then we use this function in showCameraPermissionsAlert func

Reply:
Do you want something like this?
Under src/lib/updateLastScreen
- index.ts -  will have empty updateLastScreen function
- index.ios.ts - will have updateLastScreen function to save last screen

and I updateLastScreen will be called from showCameraPermissionsAlert

Reply:
@ravindra-encoresky  Yes. We should avoid making duplicated code when creating new [FileUtils.ts](https://github.com/Expensify/App/pull/47222/files/77e195af1b7ccdcc2ec3de3676a7b7683e3ae561#diff-7b7ac9bf96ec957e26bc969984cf506e0b4af2a4b1d58611dd66111be49227f4)